layouts/shortcodes/interactive_panels.html

```diff
@@ -0,0 +1,10 @@
+{{ $firstItems := .Get "first" }}
+{{ $items := cond (eq (.Get "items") "integrations") .Site.Data.companies.integrations .Site.Data.companies.pro_services }}
```

Comment:
Is there a reason we're passing in what's essentially a bool and then loading them? It seems like it would be better to just pass in the list itself

Reply:
I have replaced the bool with an actual list item in the `.Site.Data.companies` file.